feature: PausableComposition feature flag in CCGP (#4546)
JetBrains/kotlin-web-site: docs/topics/compose-compiler-options.md

>
{style="warning"}

### PausableComposition

> > The feature flag affects behavior only with a version of Compose runtime that supports pausable composition,

Reviewer: Why not combine the info and warning into one warning at the top? If the warning is at the bottom of the text then readers can miss it if they don't read through all the content. Or think the warning is applicable to the next section. WDYT? :)

> starting with `androidx.compose.runtime` 1.8.0-alpha02.

> > With older versions, the feature flag has no effect.

Reviewer: ```suggestion
> Older versions ignore the feature flag.
```

>
{style="note"}

**Default**: disabled

> If enabled, changes code generation of composable functions to enable pausing when part of a pausable composition.

Reviewer: ```suggestion
If enabled, changes the code generation of composable functions to allow pausing when part of a pausable composition.
```

> This allows Compose runtime to suspend composition on the skipping points,
> letting long-running compositions be split across multiple frames.

Reviewer: ```suggestion
This lets Compose runtime suspend composition at skipping points,
splitting long-running compositions across multiple frames.
```

> Pausable composition is used in Lazy lists and other performance intensive components for prefetching content

Reviewer: ```suggestion
Lazy lists and other performance intensive components use pausable composition to prefetch content
```
